CHANGELOG.md

```diff
@@ -701,6 +703,17 @@ filename | sha256 hash
 
 [Documentation](https://docs.k8s.io) & [Examples](https://releases.k8s.io/release-1.6/examples)
 
+## Known Issues for v1.6.5
+
+* If you use the [GLBC Ingress Controller](https://github.com/kubernetes/ingress/tree/master/controllers/gce),
```

Comment:
I prefer to keep the "If" clause simple by focusing on a single filter, so it's easy to tell at a glance "should I read the rest of this?". Of all the applicable filters, I think the one with the best signal-to-noise ratio is "do you use GLBC Ingress?"